sparkle/src/components/Tabs.tsx

```diff
@@ -27,20 +27,40 @@ const TabsTrigger = React.forwardRef<
     label?: string;
     icon?: React.ComponentType;
     isLoading?: boolean;
+  } & Omit<LinkWrapperProps, "children" | "className">
+>(
+  (
+    { className, label, icon, href, target, rel, replace, shallow, ...props },
+    ref
+  ) => {
+    const content = (
+      <TabsPrimitive.Trigger
+        ref={ref}
+        className={cn(
+          "s-border-0 s-border-b-2 s-border-primary-800/0 s-pb-1 disabled:s-pointer-events-none data-[state=active]:s-border-primary-800",
+          className
+        )}
+        {...props}
+        asChild={!!href}
+      >
+        <Button variant="ghost" size="sm" label={label} icon={icon} />
+      </TabsPrimitive.Trigger>
+    );
+
+    return (
+      <LinkWrapper
```

Comment:
Looks tempting but won't this add one more layer/fragment in the Tree?

Reply:
It does indeed, but not sure we have other options here?

Reply:
Let's keep it like this for now then.